App.vue

```diff
@@ -74,4 +74,9 @@ body {
     }
   }
 }
+
+.sf-input input, .sf-search-bar__input {
```

Comment:
move styling to search bar component.
[https://github.com/vuestorefront/vsf-capybara/blob/develop/components/organisms/o-search.vue](url)